### DISCUSSION TITLE
Contributing to Voiceliner

### DISCUSSION THREAD

**Discussion by DarthPleurotus** [maxkrieger/voiceliner#84] [General]
Hi Max,

First off I just want to say that I discovered Voiceliner yesterday and I think it is the best software tool I've seen in months, maybe years. It simply makes sense as a tool designed for human use in a way that many more complex and more intensely developed apps do not.

You've expressed a desire for/openness to assistance with improving the app. I don't know much at all about mobile app development but I am very motivated to use my free time to help. If you could point me towards some learning resources that could get me started on understanding the relevant concepts, I would love to try to tackle some issues for you.

[For reference, I have a basice understanding of coding (I've written simple programs/scripts in Python and JS) and am reasonably tech-literate-- but no experience in mobile development]

**Comment by maxkrieger** [comment]
I appreciate the kind words! The app is entirely written in Flutter, so learning resources around that will take you in the right direction.

Some resources are here https://flutter.dev/learn 

I usually tag issues that are easier here https://github.com/maxkrieger/voiceliner/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22 - unfortunately a lot of issues  tagged "bug" are outside my own expertise.

Feel free to use this thread for further questions!

**Comment by DarthPleurotus** [comment]
Thanks! I will begin checking out Flutter in my free time.
